charts/mlflow/templates/deployment.yaml

```diff
@@ -66,22 +66,26 @@ spec:
             - name: http
               containerPort: 5000
               protocol: TCP
+          {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-            {{- if .Values.extraArgs.staticPrefix }}
-              path: {{ .Values.extraArgs.staticPrefix }}
-            {{- else }}
-              path: /
-            {{- end }}
+              path: {{ .Values.livenessProbe.healthCheckPath default "/" }}
               port: 5000
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds | default 30 }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds | default 20 }}
+            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds | default 6 }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold | default 3 }}
```

Comment:
The default configuration is a bit useless here. Because we already have a default value coming from the `values.yaml` file. This is the same for all upper and following change lines.